recipes-kernel/linux/files/bcm2712-raspberrypi5-usb.dtso

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2024 EPAM systems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+*/
+/dts-v1/;
+/plugin/;
+
+&rp1 {
```

Comment:
add here to avoid build warnings/errors:
```

#address-cells = <0x2>;
#size-cells = <0x2>;
```

also add `interrupt-parent = <&rp1>;`

Reply:
`interrupt-parent = <&rp1>;` presented in `bcm2712-raspberrypi5-domd.dts`